tests/phpunit/tests/ajax/CropImage.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+/**
+ * Admin Ajax functions to be tested.
+ */
+require_once ABSPATH . 'wp-admin/includes/ajax-actions.php';
+
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-base.php';
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-direct.php';
+
+/**
+ * Class for testing ajax crop image functionality.
+ *
+ * @group ajax
```

Comment:
As this test class is specific for one function, the `@covers` annotation can be set at the class level rather than at the test method level.

```suggestion
 * @group ajax
 * @covers ::wp_ajax_crop_image
```

Reply:
Good to know. Thank you.
Fixed in 412e86234bf99b738912fecd5d84be4897a4eb30